### ISSUE TITLE
Can't change workspace with french keymap

### ISSUE THREAD

**Issue by leeroyke** [swaywm/sway#2054]
Hi,
I set my keymap with create a file sway-fr.desktop and /usr/bin/sway-fr (I use GDM)

The file /usr/bin/sway-fr contain this 

```
#!/bin/sh
XKB_DEFAULT_LAYOUT=fr XKB_DEFAULT_VARIANT=,nodeadkeys XKB_DEFAULT_MODEL=pc102 sway
```

When i start sway with GDM, the french keymap is correct, but i can't change workspace with $mod+2 3 4 etc...

I use this version on Fedora 27

`sway version  (2017-12-30, branch "")`

Thanks in advance.

**Comment by rassouly** [comment]
Hi,
I also use a French keyboard and I use the following to change workspace.
```
bindsym Mod4+ampersand workspace 1
bindsym Mod4+eacute workspace 2
bindsym Mod4+quotedbl workspace 3
bindsym Mod4+apostrophe workspace 4
bindsym Mod4+parenleft workspace 5
bindsym Mod4+minus workspace 6
bindsym Mod4+egrave workspace 7
bindsym Mod4+underscore workspace 8
bindsym Mod4+ccedilla workspace 9
bindsym Mod4+agrave workspace 10
```
I can't recall how I found the names though.

**Comment by SirCmpwn** [comment]
I'm closing issues that are resolved by the yet-unrelated sway 1.0.
